contribute.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Standup",
+    "description": "Web app that logs daily standup updates",
+    "repository": {
+        "url": "https://github.com/mozilla/standup",
+        "license": "BSD-3-Clause",
+        "tests": "https://travis-ci.org/mozilla/standup/"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/Standup",
```

Comment:
So this is the Mozilla-centric page covering how we're running Standup, but it's not a page that's relevant to the Standup product which is in this repo. I'm not sure how to disentangle Standup the product vs. Standup the Mozilla service. This is a problem I have with `contribute.json` in general.

If someone else were to stand up a Standup, we'd end up with all the support requests unless they stomped on this file somehow which is tricky unless they fork the repo ... Messy.